packages/react-search-ui-views/src/styles/themes/reference-ui/layouts/_layout.scss

```diff
@@ -2,67 +2,143 @@
   width: 100%;
   display: flex;
   flex-direction: column;
+}
+
+/**
+ * Header / Searchbox
+ */
+
+@include block("layout-header") {
+  padding: 32px 24px;
+  border-bottom: 1px solid #eeeeee;
+}
+
+/**
+ * Body
+ */
 
-  @include block("layout-header") {
-    padding: 32px 24px;
-    border-bottom: 1px solid #eeeeee;
+@include block("layout-body") {
+  background: #fcfcfc;
+
+  &:after {
+    content: "";
```

Comment:
What are your thoughts on alphabetizing properties? Is there another strategy you use here to keep properties organized? I usually highlight the same-level properties after writing them, and just get the text editor the alphabetize them.

Reply:
For this line specifically I don't believe I touched in the CSS itself, just un-nested/un-indented it - but in general, I really prefer @mdo's [Code Guide](https://codeguide.co/#css-declaration-order) which orders by positioning, box model, then typography/visual, etc. This helps highlight properties that can massively impact how CSS displays (e.g. z-index, display, position) while keeping properties grouped that make sense together (e.g. flex properties). The full property order list is [here](https://github.com/twbs/stylelint-config-twbs-bootstrap/blob/master/css/index.js#L54) (which I believe is based on/originated from Twitter's Recess library).

Reply:
Interesting! I've always had trouble with how bootstrap organized properties, but that might be because I've never seen this. I'll look into it some more!